documentation/config_file_puppetserver.markdown

```diff
@@ -54,6 +54,8 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 
         JRuby flushing can be useful for working around buggy module code that would otherwise cause memory leaks, but it slightly reduces performance whenever a new JRuby instance reloads all of the Puppet Ruby code. If memory leaks from module code are not an issue in your deployment, the default value of 0 performs best.
 
+    * `multithreaded`: Optional, false by default. Configures Puppet Server to use a single JRuby instance to process requests that require a JRuby, processing a number of threads up to `max-active-instances` at a time. Reduces the memory footprint of the server by only requiring a single JRuby. When you configure this setting, you must also set Puppet's `multithreaded` setting to `true`.
```

Comment:
I don't remember putting a `multithreaded` setting into Puppet. Where is that?

Reply:
Oh, maybe we didn't? Oops. I guess we're mostly just checking if we run in JRuby?

Reply:
Yeah, everything should just use the threadsafe implementation if it's running on java

Reply:
Fixed. Thanks for catching that.